test/chaos/ci/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+FROM ubuntu:bionic
+
+RUN apt-get update && apt-get -qy install \
+    ca-certificates \
+    wait-for-it
```

Comment:
Do we need ca-certificates?

I'd rather we stop installing `wait-for-it` into our containers -- either the service itself should be fine continuously retrying until its dependencies are ready, or we should use the orchestrater (e.g. mzconduct) to make sure it doesn't come up until it's ready.

Reply:
Yes good point!